src/libs/logos.js

```diff
@@ -23,15 +23,15 @@ import { isAnvil, isBaseline, isBioDataCatalyst, isDatastage, isElwazi, isFirecl
 import * as Utils from 'src/libs/utils'
 
 
-export const getAppName = (longName = false) => Utils.cond(
+export const getAppName = ({ longName = false, capitalInitial = false } = {}) => Utils.cond(
   [isFirecloud(), () => 'FireCloud'],
   [isDatastage(), () => 'DataStage'],
   [isAnvil(), () => longName ? 'The NHGRI AnVIL (Genomic Data Science Analysis, Visualization, and Informatics Lab-space)' : 'AnVIL'],
   [isBioDataCatalyst(), () => 'NHLBI BioData Catalyst'],
   [isBaseline(), () => longName ? 'The Baseline Health Study Data Portal' : 'Project Baseline'],
   [isElwazi(), () => longName ? 'The eLwazi Open Data Science Platform' : 'eLwazi'],
   [isProjectSingular(), () => 'Project Singular'],
-  [isRareX(), () => longName ? 'The Rare Disease Database Platform' : 'RARE-X'],
+  [isRareX(), () => `${capitalInitial ? 'The' : 'the'} RARE-X Data Analysis Platform`],
```

Comment:
Replacing the hyphen here with a non-breaking hyphen (`\u2011`) would prevent this:

![Screen Shot 2022-07-12 at 11 33 10 AM](https://user-images.githubusercontent.com/1156625/178528551-b9a062ed-b9ae-46ad-9b59-8c2b3f93384d.png)

Reply:
Good catch! Done in [1b88e00](https://github.com/DataBiosphere/terra-ui/pull/3212/commits/1b88e00ddeaea844736149eb8b423bca06acecda).